homeassistant/components/switchbot_cloud/__init__.py

```diff
@@ -153,7 +154,10 @@ async def make_device_data(
         )
         devices_data.vacuums.append((device, coordinator))
 
-    if isinstance(device, Device) and device.device_type.startswith("Smart Lock"):
+    if isinstance(device, Device) and device.device_type in [
```

Comment:
it's bug here, my App added new device[Lock Ultra], this device 's device_type also start with Smart Lock, but in sensor.py it has no description for Lock Ultra, so it raise error